tests.py

```diff
@@ -20,3 +23,26 @@ def test_allows_locally(self):
 
     def test_errors_on_remote(self):
         self.assertRaises(SocketError, lambda: urlopen('http://www.disqus.com'))
+
+
+class Options(object):
+    socket_trace = True
+
+
+class ReportingSocketWhitelistPluginTestCase(unittest.TestCase):
```

Comment:
Nice test.

Did you mean for this to be called `LoggingSocketWhitelistPluginTestCase`?

Reply:
yes LoggingSocketWhitelistPluginTestCase is what it should be.